ofono/bridge.py

```diff
@@ -0,0 +1,134 @@
+import logging
+import os
+from time import sleep
+
+import pydbus
+from dbus.mainloop.glib import DBusGMainLoop
+from gi.repository import GLib
+
+from helpers import Singleton, setup_logger
+
+DBusGMainLoop(set_as_default=True)  # has to be called first
+
+logger = setup_logger(__name__, 'debug')
+logging.basicConfig(level=logging.DEBUG)
+
+
+class OfonoBridge(object):
+    """
+    Generic util class to bridge between ZPUI and ofono backend through D-Bus
+    """
+
+    def __init__(self):
+        super(OfonoBridge, self).__init__()
```

Comment:
Wait, it doesn't inherit from anything =D

Reply:
... yet :)